export_database.py

```diff
@@ -0,0 +1,101 @@
+from pymongo import MongoClient
+from pymongo import errors
+from os import environ as env
+import bson
+from bson.json_util import dumps, loads
+
+def export_db():
+    '''
+        Tries to Exports the whole Database (each and every notice)
+        Saves notices already in target Database in repeated_notices.bson
+        Saves notices failed to be inserted in defaulters.bson
+
+        Can be run multiple times with no fear of multiple entries in case of any confusion
+    '''
+    print("connecting to mlab DB")
+    mc_mlab = MongoClient(env['MONGODB_URI'])
+    print("collecting notices cursor from MLab")
+    notices_cursor = mc_mlab.get_default_database().notices.find()
+    print("connecting to atlas DB")
+    mc_atlas = MongoClient(env['MONGODB_URI_ATLAS'])
```

Comment:
Can we generalize this migration script? If we need to do a migration in the future we may need it again.

A suggestion would be the naming the env variable as OLD_XXXX and NEW_XXXX instead of ATLAS and MONGO. The script may require other changes as well for being general and I think you can find those better.

Reply:
Okay if we're going to merge this then I'll change it properly!